public/javascripts/Admin/src/Admin.js

```diff
@@ -301,6 +301,134 @@ function Admin(_, $) {
                 vega.embed("#completion-progress-chart", chart, opt, function(error, results) {});
             });
 
+            $.getJSON('/adminapi/labelTags', function(data) {
+                var curbRamps = data.filter(function(label) {return label.label_type === "CurbRamp"});
```

Comment:
There has to be a more space-efficient way to write these 50 lines or so... I'm thinking probably:
* One line to remove any label types we don't want to plot (Optional)
* Shortening the tags only has to happen once (though you could have done this anyway by running it on `data` before splitting it)
* Then when you want to use the data from a specific label type for a plot, the line would probably look like:
    ```
    "data": {"values": data.filter(function(label) {return label.label_type === "CurbRamp"})}
    ```
* Side note: I'd rename `data` to `tagCounts` or `tagCountData`. Parameters were named poorly in the past, but we don't have to continue the trend :grin: